Bottom: c07656981b5e9d7ee8dbffb163a573904ede3f53
Top:    de5030388d3a0f9325c63bd934f6ab0fc13903e4
Author: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>
Date:   2020-10-14 18:11:43 +0300

allow for armv7 soft and hard float in bsp tool

Signed-off-by: Robert Berger <robert.berger@ReliableEmbeddedSystems.com>


---

diff --git a/scripts/lib/bsp/substrate/target/arch/arm/conf/machine/machine.conf b/scripts/lib/bsp/substrate/target/arch/arm/conf/machine/machine.conf
index 36d0a25df8..4f517cce92 100644
--- a/scripts/lib/bsp/substrate/target/arch/arm/conf/machine/machine.conf
+++ b/scripts/lib/bsp/substrate/target/arch/arm/conf/machine/machine.conf
@@ -35,7 +35,8 @@ EXTRA_IMAGEDEPENDS += "u-boot"
 {{ input type:"choice" val:"tune_iwmmxt" msg:"iwmmxt tuning optimizations" }}
 {{ input type:"choice" val:"tune_strongarm1100" msg:"strongarm1100 tuning optimizations" }}
 {{ input type:"choice" val:"tune_xscale" msg:"xscale tuning optimizations" }}
-{{ input type:"choice" val:"tune_armv7a" msg:"arch-armv7a+armv7a-neon" }}
+{{ input type:"choice" val:"tune_armv7a_soft_float" msg:"arch-armv7a+armv7a-neon" }}
+{{ input type:"choice" val:"tune_armv7a_hard_float" msg:"arch-armv7a+armv7athf" }}
 {{ if tunefile == "tune_arm1136jf_s": }}
 include conf/machine/include/tune-arm1136jf-s.inc
 {{ if tunefile == "tune_arm920t": }}
@@ -69,9 +70,12 @@ include conf/machine/include/tune-iwmmxt.inc
 include conf/machine/include/tune-strongarm1100.inc
 {{ if tunefile == "tune_xscale": }}
 include conf/machine/include/tune-xscale.inc
-{{ if tunefile == "tune_armv7a":}}
+{{ if tunefile == "tune_armv7a_soft_float":}}
 DEFAULTTUNE = "armv7a-neon"
 include conf/machine/include/arm/arch-armv7a.inc
+{{ if tunefile == "tune_armv7a_hard_float":}}
+DEFAULTTUNE = "armv7athf"
+require conf/machine/include/arm/arch-armv7a.inc
 
 IMAGE_FSTYPES += "tar.bz2"
